diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/EmptyBlock.kt

```diff
@@ -70,6 +77,15 @@ class EmptyBlock(private val configRules: List<RulesConfig>) : Rule("empty-block
         }
     }
 
+    @Suppress("UnsafeCallOnNullableType")
+    private fun isAnonymousSAMClass(node: ASTNode) : Boolean {
+        return if (node.elementType == FUNCTION_LITERAL && node.hasParent(VALUE_ARGUMENT)) {
+            val valueArgument = node.findParentNodeWithSpecificType(VALUE_ARGUMENT)!!
+            valueArgument.findLeafWithSpecificType(IDENTIFIER)?.text?.isPascalCase() ?: false
```

Comment:
What does this check? How does identifier case relate to it's meaning?

Reply:
This value argument is not a class in AST, so the idea is to check identifier naming